src/lookup-table/models/LookupTable.model.ts

```diff
@@ -2,6 +2,12 @@ import { LookupTableStructurePropType } from '../types'
 import DBConnect from '../../components/databaseConnect'
 
 const LookupTableModel = () => {
+  const getAll = async ({ tableName }: any) => {
+    const text = `SELECT * FROM lookup_table_${tableName}`
```

Comment:
I think it's neater to use fieldMap here instead of in controller:
```JS
fields = fieldMap.map(({label, fieldname}) => `"${fielname}" as "${label}"`).join(',')
`SELECT ${fields} FROM lookup_table_${tableName}`
```

Reply:
This has already been done by @andreievg and is in `master`